---
sidebar: auto
sidebarDepth: 2
---


# DeepTools-Microbiome
_A list of deeplearning tools for metagenome data analysis Whereas most of them are Open Source, not all of them are. Please check licences that may apply.This list contains items that i **personally** find useful, usages may differ._

Please feel free to comment or open a pull request if you know better tools / or alternatives or if some links are dead, ... ðŸ˜‰

## Identifier for viral sequences from metagenome 
*Planning is half the work*

* [Virtifier](https://github.com/crazyinter/Seq2Vec) - [Virtifier: a deep learning-based identifier for viral sequences from metagenomes](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab845/6462188?redirectedFrom=fulltext)
* [DeepVirFinder](https://github.com/jessieren/DeepVirFinder) - [DeepVirFinder: Identifying viruses from metagenomic data by deep learning](https://link.springer.com/article/10.1007/s40484-019-0187-4)
* [PPR-Meta](https://github.com/zhenchengfang/PPR-Meta) -[PPR-Meta: a tool for identifying phages and plasmids from metagenomic fragments using deep learning](https://academic.oup.com/gigascience/article/8/6/giz066/5521157)

## Aseembly and bining 

* [VAMB](https://github.com/RasmussenLab/vamb) - [Improved metagenome binning and assembly using deep variational autoencoders](https://www.nature.com/articles/s41587-020-00777-4?proof=t%25C2%25A0)



## Biosynthetic Gene cluster detection and classification 

* [DeepBGC](https://github.com/Merck/deepbgc) -[https://academic.oup.com/nar/article/47/18/e110/5545735](https://academic.oup.com/nar/article/47/18/e110/5545735)
